server/lib/pbench/indexer.py

```diff
@@ -2013,30 +1982,30 @@ def _make_source_unified(self):
         # by deriving data from the header rows of all the csv files,
         # first. This is driven by the data provided in the handler.
         self.logger.info("tool-data-indexing: tool {}, start unified for {}", self.toolname, self.basepath)
-        for csv in self.files:
+        for csvf in self.files:
```

Comment:
good catch here

Reply:
Yeah, this was very subtle and a big mistake on my part when I wrote this.